### DISCUSSION TITLE
OpenGitOps Open Forum

### DISCUSSION THREAD

**Discussion by christianh814** [open-gitops/project#121] [Ideas]
I have had numerous of a̶r̶g̶u̶m̶e̶n̶t̶s̶  discussions (in person and online) about GitOps, best practices, and use cases. Usually I try to lead people to https://opengitops.dev/get-involved.

**Issue**: The Fort-weekly meeting is just not enough time for these types of discussions. By the time we get through updates and issues there isn't enough time to have a meaningful discussion.

**Proposal**: Have semi-regular "meetup" style event called **_"OpenGitOps Open Forum"_**. This would be "birds of a feather" type of meetup where we can have long form discussions either in "break out session" style or as a whole group (I like the break out session style as there will be a LOT of topics to be discussed about). I say we try this out virtually (using something like hopin.com) with the hope of doing some in-person meetups?

**What this is NOT**: This isn't meant to replace GitOpsCon, on the contrary, it's meant to supplement it. The "Open Forum" will **not** have a CFP process or anything like that. It's all free form in the same spirit as "birds of a feather". It should be ALL community driven - only sponsorship should be if a company is providing the venue/platform (either virtual or in-person). 

**End Result**: The end result should be drafts of best practices, white papers, and/or PRs to update things. I see, as the only rule, that everything must conform to the GitOps Principles v1.

I've discussed this with @scottrigby but would like others to chime in as well.

**Comment by williamcaban** [comment]
That's a great idea! Such forum could also help moving some PRs of topics waiting for discussions.

**Comment by niklasmtj** [comment]
Yes please! This sounds like a great thing. Especially the in person meetups (even when they're probably not near me haha). I'm currently trying to organise a community meetup here in Cologne. More general about DevOps / SRE topics but GitOps is and will always be very welcome to be discussed.

Really looking forward for such a format of an open forum! Count me in for help with the organization. I think we can talk about current topics more easily and more in depth than in the group meetings. Especially in regard of the sustainability whitepaper this format can be a great way to talk about the details and get new ideas.

**Comment by kmf** [comment]
I think people would love to have a platform where they can discuss their thoughts, this will make the **_principles_** more tangible, so people would understand the "heart" behind GitOps. This in turn will refine the better understanding of the practices. (the why /how)

**Comment by scottrigby** [comment]
I love it @christianh814 ❤️ I realize I had not weighed in on this discussion thread here. You know this is what I was initially wanting to see as part of GItOpsCon, but it posed logistical challenges - however what you propose above sounds realistic! Given that [GItOpsCon is moving to Open Source Summit this year](https://opengitops.dev/blog/cdcon+gitopscon-at-open-source-summit/), and that we still have an opportunity to do a one-day or half-day event or project meeting at KubeCon, perhaps we should revisit this now!

**Comment by fharper** [comment]
Is this initiative started? Or does someone own this already?

**Reply by niklasmtj** [reply]
We would like to try this out as part of the our meeting at KubeCon EU in Amsterdam. But beyond that this is still unstructured. I don't think that Someone really owns it at the moment. Maybe @christianh814 can say something about that. 

See [#142](https://github.com/open-gitops/project/discussions/142) for the poll what we would like to do at KubeCon EU

**Reply by fharper** [reply]
OK, that could be something I would be interested to help, for outside of the next KubeCon EU as I can't be there this year.
